CONTRIBUTING.md

```diff
@@ -284,13 +284,13 @@ Various hints for developers
 - We are using codecov which has extensions for the popular browsers
   (Firefox, Chrome) which annotates pull requests on github regarding changed coverage.
 
-### Useful Flags
+### Useful Environment Variables
+Refer datalad/config.py for information on how to add these environment variables to the config file and their naming convention
```

Comment:
I wish we were there ;-) but let's aim for it indeed, configuration refactoring is coming!!! ;-)